db_revisions/versions/6613e1e2c133_replace_is_active_with_lei_status_in_fi.py

```diff
@@ -66,17 +70,23 @@ def upgrade() -> None:
         batch_op.drop_column("is_active")
 
     # Removing is_active from and adding lei_status_code to financial_institutions_history table
-    op.drop_column("financial_institutions_history", "is_active")
     op.add_column("financial_institutions_history", sa.Column("lei_status_code", sa.String()))
 
 
 def downgrade() -> None:
 
     op.drop_column("financial_institutions_history", "lei_status_code")
-    op.add_column("financial_institutions_history", sa.Column("is_active", sa.Boolean()))
+
+    op.add_column(
+        "financial_institutions",
+        sa.Column(name="is_active", type_=sa.Boolean(), nullable=True, server_default=sa.true()),
+    )
```

Comment:
If you're providing a server_default, nullable can be false (or just remove that nullable=True) and then remove the alter_column below.